docs/pages/Builder.md

```diff
@@ -68,7 +68,7 @@ All available standard tasks are documented [in the API reference](https://sap.g
 | generateBundle            | *disabled* ^4^     | *disabled* ^4^ |                      |
 | buildThemes               |                    | *enabled*      | *enabled*            |
 | generateThemeDesignerResources |               | *disabled* ^5^ | *disabled* ^5^       |
-| generateVersionInfo       | *disabled*         |                |                      |
+| generateVersionInfo       |                    | *disabled* ^1^ |                      |
```

Comment:
I'd expect this to be in the "application" column, not in the library column, right?

Reply:
I have added the term "disabled" to all columns because its relevant for any jsdoc build triggered independent of the kind of project, isn't?

Reply:
Not exactly. It can't be activated in other project types, so it shouldn't be listed there. Just for application with the ^1^ remark.

Reply:
Oh I was not aware of that. Sure. will change that